paddle/infrt/dialect/infrt/infrt_ops.td

```diff
@@ -0,0 +1,52 @@
+#ifndef Infrt_OpS
+#define Infrt_OpS
+
+include "mlir/IR/OpBase.td"
+include "mlir/Interfaces/SideEffectInterfaces.td"
+
+def Infrt_Dialect : Dialect {
+  let summary =
+    "A dialect containing the Infrt Attributes, Operations, and Types";
+
+  let name = "Infrt";
```

Comment:
infrt ? 

It is better to replace Infrt_Dialect with  INFRT_Dialect, to keep consistent with the other places.

Reply:
ok